samples/sensor/icp101xx/README.rst

```diff
@@ -0,0 +1,49 @@
+.. _icp101xx:
+
+ICP101xx: Barometric Pressure and Temperature Sensor
```

Comment:
We have a dht_polling sample that covers a similar case, I don't think we need a device specific sample in this scenario

Reply:
DHT is a sensor from another manufacturer (Aosong Electronics) and is NOT a pressure sensor, this sample can definitely not be used with the TDK ICP101xx pressure sensor.